roles/openshift_prometheus/tasks/install_prometheus.yaml

```diff
@@ -0,0 +1,223 @@
+---
```

Comment:
I don't think this is a sustainable pattern.  This means we are defining config twice - once in origin that uses core kube objects, and once here.  I don't think we should be turning core origin config into roles like this.

I'd like the config to be owned by the team - to be provided as a set of core yaml to ansible.  I'd then like ansible to be customized as necessary for the supported outcomes.  I do not like recreating the config in total in another form for Ansible - that's unsustainable in the long run.  The core config in OpenShift is our supported platform.

@liggitt @sdodson this was our discussion today - I do not want to see this pattern continued, because it is not leveraging the platform as it is intended to be used.